README.md

```diff
@@ -15,12 +15,11 @@ TODO
 
 ## Usage
 
-Using Gidari is a two step process:
+Document how to use the package with `gidari.Transport` 
```

Comment:
We don't want to add this line exactly, we need to document the transport use case. At the moment, only `TransportFile` is relevant. You'll have to add a code block here to show how to use that function.